Develop

Shiuan-Chang opened the pull request:
Hi cwpeng,

My name is 張瑞璇, my discord name is RSChang.
The following link is my task this week. If you have any suggestion, please do not hesitate to tell me.

http://34.225.66.108:3000/

Thanks!

cwpeng reviewed (changes requested):
1. We should redirect user to homepage if user visits booking page without signing in.
2. I cannot add a booking.

Shiuan-Chang commented:
Hi Cwpeng,

Thank you for your feedback. I've addressed the issues you highlighted earlier. Specifically, for the second part, I've turned off the debug mode in the app.py file, resolving the issue. You can review the updated version here:

http://34.225.66.108:3000/

Should you encounter any other concerns or require further modifications, please feel free to let me know.

Regards
瑞璇

cwpeng reviewed (approved):
OK. Done.
